kubernetes_deploy/staging/queue_worker_deployment.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        worker: laa-legal-adviser-api-worker
+        env: staging
+        service_area: laa-get-access
+        service_team: cla-fala
+    spec:
+      containers:
+      - image: "<to be set by deploy_to_kubernetes>"
+        name: worker
+        args: ["/home/app/docker/run_worker.sh"]
```

Comment:
Curious, I thought `WORKDIR /home/app` would negate the need for full path here. Do we need it?

Reply:
👋 Bumping this question. Is it expected that relative paths do not work?